lib/resources/r4/communication.yaml

```diff
@@ -180,5 +164,45 @@ fields:
         }
       ]
     }
-  action:
-    - create
+  action: create
+
+- name: Communication Email-Status Extension
+  terminology_name: extension[x].valueCodeableConcept
+  required: 'No'
+  type: CodeableConcept
+  description: The status of the email. May be received, in-progress or completed.
+  action: create
```

Comment:
add terminology. I think Provenance does both

Reply:
This needs to be just in @yhcerner PR. This is only for terminology. Dont need the example

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-f08ae1e4c063b9a36a22adaa43fecf21L188-L197

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-f08ae1e4c063b9a36a22adaa43fecf21L188-R186

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/files#diff-f08ae1e4c063b9a36a22adaa43fecf21L188-R186